assets/src/setup/pages/summary/desktop-screenshot.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { Desktop } from '../../components/desktop';
+
+export function DesktopScreenshot( { screenshot, name, description, url } ) {
```

Comment:
Copying over my comment from the Slack channel:

> Seeing the active theme and its description randomly on the summary page seems out of place to me, since the user did not see it as during the setup process (except for reader mode). Adding a label saying the active theme is "X" and maybe quoting the theme's description could make it better.

> ![image](https://user-images.githubusercontent.com/16200219/85819312-cd769480-b762-11ea-8eb3-5bf1801e85cc.png)